{"uid":"d7b64513c861fc40","name":"Тест загрузки МК","fullName":"tests.test_calculation_constructor.TestCalculationConstructor#test_parser","historyId":"1f002515f30fbf7c7f659df4415c1ef0","time":{"start":1676343699798,"stop":1676343784202,"duration":84404},"description":"Тест запустил - Данил Подольный \nСтенд, на котором запускался тест - http://genrestest.nntc.pro/economics/","descriptionHtml":"<p>Тест запустил - Данил Подольный<br />\nСтенд, на котором запускался тест - http://genrestest.nntc.pro/economics/</p>\n","status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\dpodolny\\\\PycharmProjects\\\\IIFA_Economics_Auto-tests_v.0.3\\\\video_reports\\\\test_parser[\\\\u041c\\\\u041a_Upstream_v22.6\\\\u0411_6+6_\\\\u042e\\\\u0423\\\\u041d\\\\u0413_\\\\u0411\\\\u0443\\\\u0440\\\\u0435\\\\u043d\\\\u0438\\\\u0435_3_51]_10_01_34.mp4'","statusTrace":"self = <allure_commons._allure.fixture object at 0x000002090740C590>, args = ()\nkwargs = {}\n\n    def __call__(self, *args, **kwargs):\n        self.parameters = func_parameters(self._fixture_function, *args, **kwargs)\n    \n        with self:\n>           return self._fixture_function(*args, **kwargs)\n\n..\\..\\..\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\allure_commons\\_allure.py:221: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\conftest.py:57: in record_video\n    allure.attach.file(outfile, attachment_type=allure.attachment_type.MP4)\n..\\..\\..\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\allure_commons\\_allure.py:203: in file\n    plugin_manager.hook.attach_file(source=source, name=name, attachment_type=attachment_type, extension=extension)\n..\\..\\..\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\allure_pytest\\listener.py:233: in attach_file\n    self.allure_logger.attach_file(uuid4(), source, name=name, attachment_type=attachment_type, extension=extension)\n..\\..\\..\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\allure_commons\\reporter.py:160: in attach_file\n    plugin_manager.hook.report_attached_file(source=source, file_name=file_name)\n..\\..\\..\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\allure_commons\\logger.py:53: in report_attached_file\n    shutil.copy2(source, destination)\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\shutil.py:436: in copy2\n    copyfile(src, dst, follow_symlinks=follow_symlinks)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nsrc = 'C:\\\\Users\\\\dpodolny\\\\PycharmProjects\\\\IIFA_Economics_Auto-tests_v.0.3\\\\video_reports\\\\test_parser[\\\\u041c\\\\u041a_Upstream_v22.6\\\\u0411_6+6_\\\\u042e\\\\u0423\\\\u041d\\\\u0413_\\\\u0411\\\\u0443\\\\u0440\\\\u0435\\\\u043d\\\\u0438\\\\u0435_3_51]_10_01_34.mp4'\ndst = 'C:\\\\Users\\\\dpodolny\\\\PycharmProjects\\\\IIFA_Economics_Auto-tests_v.0.3\\\\tests\\\\allure-results\\\\d3d3bcc1-84f8-4c27-a1fe-e498c8d9f631-attachment.mp4'\n\n    def copyfile(src, dst, *, follow_symlinks=True):\n        \"\"\"Copy data from src to dst in the most efficient way possible.\n    \n        If follow_symlinks is not set and src is a symbolic link, a new\n        symlink will be created instead of copying the file it points to.\n    \n        \"\"\"\n        sys.audit(\"shutil.copyfile\", src, dst)\n    \n        if _samefile(src, dst):\n            raise SameFileError(\"{!r} and {!r} are the same file\".format(src, dst))\n    \n        file_size = 0\n        for i, fn in enumerate([src, dst]):\n            try:\n                st = _stat(fn)\n            except OSError:\n                # File most likely does not exist\n                pass\n            else:\n                # XXX What about other special files? (sockets, devices...)\n                if stat.S_ISFIFO(st.st_mode):\n                    fn = fn.path if isinstance(fn, os.DirEntry) else fn\n                    raise SpecialFileError(\"`%s` is a named pipe\" % fn)\n                if _WINDOWS and i == 0:\n                    file_size = st.st_size\n    \n        if not follow_symlinks and _islink(src):\n            os.symlink(os.readlink(src), dst)\n        else:\n>           with open(src, 'rb') as fsrc:\nE           FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\dpodolny\\\\PycharmProjects\\\\IIFA_Economics_Auto-tests_v.0.3\\\\video_reports\\\\test_parser[\\\\u041c\\\\u041a_Upstream_v22.6\\\\u0411_6+6_\\\\u042e\\\\u0423\\\\u041d\\\\u0413_\\\\u0411\\\\u0443\\\\u0440\\\\u0435\\\\u043d\\\\u0438\\\\u0435_3_51]_10_01_34.mp4'\n\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\shutil.py:256: FileNotFoundError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"record_video","time":{"start":1676343694568,"stop":1676343694572,"duration":4},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false},{"name":"test_append_report_history","time":{"start":1676343694572,"stop":1676343694586,"duration":14},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false},{"name":"driver","time":{"start":1676343694586,"stop":1676343697362,"duration":2776},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false},{"name":"sign_in_to_stand","time":{"start":1676343697362,"stop":1676343699797,"duration":2435},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false}],"testStage":{"description":"Тест запустил - Данил Подольный \nСтенд, на котором запускался тест - http://genrestest.nntc.pro/economics/","status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\dpodolny\\\\PycharmProjects\\\\IIFA_Economics_Auto-tests_v.0.3\\\\video_reports\\\\test_parser[\\\\u041c\\\\u041a_Upstream_v22.6\\\\u0411_6+6_\\\\u042e\\\\u0423\\\\u041d\\\\u0413_\\\\u0411\\\\u0443\\\\u0440\\\\u0435\\\\u043d\\\\u0438\\\\u0435_3_51]_10_01_34.mp4'","statusTrace":"self = <allure_commons._allure.fixture object at 0x000002090740C590>, args = ()\nkwargs = {}\n\n    def __call__(self, *args, **kwargs):\n        self.parameters = func_parameters(self._fixture_function, *args, **kwargs)\n    \n        with self:\n>           return self._fixture_function(*args, **kwargs)\n\n..\\..\\..\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\allure_commons\\_allure.py:221: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\conftest.py:57: in record_video\n    allure.attach.file(outfile, attachment_type=allure.attachment_type.MP4)\n..\\..\\..\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\allure_commons\\_allure.py:203: in file\n    plugin_manager.hook.attach_file(source=source, name=name, attachment_type=attachment_type, extension=extension)\n..\\..\\..\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\allure_pytest\\listener.py:233: in attach_file\n    self.allure_logger.attach_file(uuid4(), source, name=name, attachment_type=attachment_type, extension=extension)\n..\\..\\..\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\allure_commons\\reporter.py:160: in attach_file\n    plugin_manager.hook.report_attached_file(source=source, file_name=file_name)\n..\\..\\..\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\allure_commons\\logger.py:53: in report_attached_file\n    shutil.copy2(source, destination)\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\shutil.py:436: in copy2\n    copyfile(src, dst, follow_symlinks=follow_symlinks)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nsrc = 'C:\\\\Users\\\\dpodolny\\\\PycharmProjects\\\\IIFA_Economics_Auto-tests_v.0.3\\\\video_reports\\\\test_parser[\\\\u041c\\\\u041a_Upstream_v22.6\\\\u0411_6+6_\\\\u042e\\\\u0423\\\\u041d\\\\u0413_\\\\u0411\\\\u0443\\\\u0440\\\\u0435\\\\u043d\\\\u0438\\\\u0435_3_51]_10_01_34.mp4'\ndst = 'C:\\\\Users\\\\dpodolny\\\\PycharmProjects\\\\IIFA_Economics_Auto-tests_v.0.3\\\\tests\\\\allure-results\\\\d3d3bcc1-84f8-4c27-a1fe-e498c8d9f631-attachment.mp4'\n\n    def copyfile(src, dst, *, follow_symlinks=True):\n        \"\"\"Copy data from src to dst in the most efficient way possible.\n    \n        If follow_symlinks is not set and src is a symbolic link, a new\n        symlink will be created instead of copying the file it points to.\n    \n        \"\"\"\n        sys.audit(\"shutil.copyfile\", src, dst)\n    \n        if _samefile(src, dst):\n            raise SameFileError(\"{!r} and {!r} are the same file\".format(src, dst))\n    \n        file_size = 0\n        for i, fn in enumerate([src, dst]):\n            try:\n                st = _stat(fn)\n            except OSError:\n                # File most likely does not exist\n                pass\n            else:\n                # XXX What about other special files? (sockets, devices...)\n                if stat.S_ISFIFO(st.st_mode):\n                    fn = fn.path if isinstance(fn, os.DirEntry) else fn\n                    raise SpecialFileError(\"`%s` is a named pipe\" % fn)\n                if _WINDOWS and i == 0:\n                    file_size = st.st_size\n    \n        if not follow_symlinks and _islink(src):\n            os.symlink(os.readlink(src), dst)\n        else:\n>           with open(src, 'rb') as fsrc:\nE           FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\dpodolny\\\\PycharmProjects\\\\IIFA_Economics_Auto-tests_v.0.3\\\\video_reports\\\\test_parser[\\\\u041c\\\\u041a_Upstream_v22.6\\\\u0411_6+6_\\\\u042e\\\\u0423\\\\u041d\\\\u0413_\\\\u0411\\\\u0443\\\\u0440\\\\u0435\\\\u043d\\\\u0438\\\\u0435_3_51]_10_01_34.mp4'\n\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\shutil.py:256: FileNotFoundError","steps":[{"name":"Прохождение авторизации на стенде","time":{"start":1676343699798,"stop":1676343699798,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false},{"name":"Переход в конструктор расчетов","time":{"start":1676343699798,"stop":1676343701161,"duration":1363},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false},{"name":"Очистка мастер-книг со стенда","time":{"start":1676343701161,"stop":1676343753246,"duration":52085},"status":"passed","steps":[{"name":"Подчет всех МК на стенде","time":{"start":1676343701180,"stop":1676343706625,"duration":5445},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false},{"name":"Удаление всех МК на стенде","time":{"start":1676343706625,"stop":1676343741791,"duration":35166},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false},{"name":"Проверка остались ли МК на странице","time":{"start":1676343741791,"stop":1676343746883,"duration":5092},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false},{"name":"F5 и повторная проверка","time":{"start":1676343746883,"stop":1676343753246,"duration":6363},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false}],"attachments":[],"parameters":[],"hasContent":true,"stepsCount":4,"attachmentsCount":0,"shouldDisplayMessage":false},{"name":"Загрузка мастер-книги на стенд","time":{"start":1676343753246,"stop":1676343784200,"duration":30954},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false}],"attachments":[{"uid":"794ea196594b306b","name":"stdout","source":"794ea196594b306b.txt","type":"text/plain","size":4228}],"parameters":[],"hasContent":true,"stepsCount":8,"attachmentsCount":1,"shouldDisplayMessage":true},"afterStages":[{"name":"driver::0","time":{"start":1676343784211,"stop":1676343786450,"duration":2239},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false},{"name":"record_video::0","time":{"start":1676343786452,"stop":1676343786455,"duration":3},"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\dpodolny\\\\PycharmProjects\\\\IIFA_Economics_Auto-tests_v.0.3\\\\video_reports\\\\test_parser[\\\\u041c\\\\u041a_Upstream_v22.6\\\\u0411_6+6_\\\\u042e\\\\u0423\\\\u041d\\\\u0413_\\\\u0411\\\\u0443\\\\u0440\\\\u0435\\\\u043d\\\\u0438\\\\u0435_3_51]_10_01_34.mp4'\n","statusTrace":"  File \"C:\\Users\\dpodolny\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\dpodolny\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\_pytest\\fixtures.py\", line 917, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\dpodolny\\PycharmProjects\\IIFA_Economics_Auto-tests_v.0.3\\conftest.py\", line 57, in record_video\n    allure.attach.file(outfile, attachment_type=allure.attachment_type.MP4)\n  File \"C:\\Users\\dpodolny\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\allure_commons\\_allure.py\", line 203, in file\n    plugin_manager.hook.attach_file(source=source, name=name, attachment_type=attachment_type, extension=extension)\n  File \"C:\\Users\\dpodolny\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\pluggy\\_hooks.py\", line 265, in __call__\n    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\dpodolny\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\pluggy\\_manager.py\", line 80, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\dpodolny\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\pluggy\\_callers.py\", line 60, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\dpodolny\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\pluggy\\_result.py\", line 60, in get_result\n    raise ex[1].with_traceback(ex[2])\n  File \"C:\\Users\\dpodolny\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\dpodolny\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\allure_pytest\\listener.py\", line 233, in attach_file\n    self.allure_logger.attach_file(uuid4(), source, name=name, attachment_type=attachment_type, extension=extension)\n  File \"C:\\Users\\dpodolny\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\allure_commons\\reporter.py\", line 160, in attach_file\n    plugin_manager.hook.report_attached_file(source=source, file_name=file_name)\n  File \"C:\\Users\\dpodolny\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\pluggy\\_hooks.py\", line 265, in __call__\n    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\dpodolny\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\pluggy\\_manager.py\", line 80, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\dpodolny\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\pluggy\\_callers.py\", line 60, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\dpodolny\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\pluggy\\_result.py\", line 60, in get_result\n    raise ex[1].with_traceback(ex[2])\n  File \"C:\\Users\\dpodolny\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\dpodolny\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\allure_commons\\logger.py\", line 53, in report_attached_file\n    shutil.copy2(source, destination)\n  File \"C:\\Users\\dpodolny\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\shutil.py\", line 436, in copy2\n    copyfile(src, dst, follow_symlinks=follow_symlinks)\n  File \"C:\\Users\\dpodolny\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\shutil.py\", line 256, in copyfile\n    with open(src, 'rb') as fsrc:\n         ^^^^^^^^^^^^^^^\n","steps":[],"attachments":[{"type":"video/mp4","size":0}],"parameters":[],"hasContent":true,"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":true}],"labels":[{"name":"feature","value":"Загрузка МК на фронт"},{"name":"story","value":"Загрузка всех имеющихся МК на стенд и проверка, что все загрузилось корректно"},{"name":"severity","value":"blocker"},{"name":"tag","value":"calculacion_constructor"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_calculation_constructor"},{"name":"subSuite","value":"TestCalculationConstructor"},{"name":"host","value":"C100119"},{"name":"thread","value":"56156-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_calculation_constructor"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"mb_name","value":"'МК_Upstream_v22.6Б_6+6_ЮУНГ_Бурение_3_51'"}],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":["calculacion_constructor"]},"source":"d7b64513c861fc40.json","parameterValues":["'МК_Upstream_v22.6Б_6+6_ЮУНГ_Бурение_3_51'"]}